src/fragments/lib/graphqlapi/existing-resources.mdx

```diff
@@ -19,7 +19,7 @@ Existing AWS AppSync resources can be used with the Amplify Libraries by referen
 ```
 
 - **API NAME**: Friendly name for the API (e.g., *api*)
-  - **endpoint**: The HTTPS endpoint of the AWS AppSync API (e.g. *https://aaaaaaaaaaaaaaaaaaaaaaaaaa.appsync-api.us-east-1.amazonaws.com/graphql*)
+  - **endpoint**: The HTTPS endpoint of the AWS AppSync API (e.g. *https://aaaaaaaaaaaaaaaaaaaaaaaaaa.appsync-api.us-east-1.amazonaws.com/graphql*). Custom domains can also be supplied here (e.g. *https://api.yourdomain.com/graphql*)
```

Comment:
Can we have a link to Appsync documentation for the format of the domains here, if possible?

Reply:
The custom domain can have any format, but has to end in `graphql`. I can reference that here.

Reply:
I'll also link to the AWS doc that will be published